setup.py

```diff
@@ -61,6 +61,13 @@
 PYTHON_BINDINGS_PATH = os.path.dirname(os.path.abspath(__file__))
 
 
+def set_pkg_config_path():
+    if not pkgconfig.exists("libprecice"):
+        os.environ["PKG_CONFIG_PATH"] = ":".join(
+            [path + "/pkgconfig" for path in os.environ["LD_LIBRARY_PATH"].split(":")]
```

Comment:
If we then assume that the user has configured `LD_LIBRARY_PATH` for dynamic linking of `libprecice` (because it is in a non-system location), we can then use that as `PKG_CONFIG_PATH` to find the library.